staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2771,6 +2771,7 @@ type PodSpec struct {
 	// DNS parameters given in DNSConfig will be merged with the policy selected with DNSPolicy.
 	// To have DNS options set along with hostNetwork, you have to specify DNS policy
 	// explicitly to 'ClusterFirstWithHostNet'.
+	// Note that 'None' policy is an alpha feature introduced in v1.9 and CustomPodDNS feature gate must be enabled to use the feature.
```

Comment:
A similar comment needs to be added to DNSConfig field, which is also gated by CustomPodDNS feature gate as well.

Reply:
ack